README.md

```diff
@@ -22,17 +54,13 @@ options:
 -p:     For upload, directory or file that should be uploaded to Wasabi  
 -s:     For upload, the server address/endpoint to upload to  
 -z:     For upload, the path to a zip file that contains security credentials for Wasabi
+```
 
 Normally these will be called from a container so you will have something like:
 
-docker run --rm -v ./local/path/to/scan:/tmp/scan -v ./local/path/to/goatrodeo/output:/tmp/output spicelabs/grinder -c adg -b /tmp/scan -0 /tmp/output
+```docker run --rm -v ./local/path/to/scan:/tmp/scan -v ./local/path/to/goatrodeo/output:/tmp/output spicelabs/grinder -c adg -b /tmp/scan -0 /tmp/output```
```

Comment:
```suggestion
```bash
docker run --rm -v ./local/path/to/scan:/tmp/scan -v ./local/path/to/goatrodeo/output:/tmp/output spicelabs/grinder -c adg -b /tmp/scan -0 /tmp/output
```
```